index.html

```diff
@@ -304,37 +304,37 @@ <h6 class="uppercase mb16">Technologies used: Kotlin, Java, Shell Script, Python
     </div>
 </section>
 <section>
-    <div class="container">
-        <div class="row text-center">
-            <div class="col-sm-4">
+    <div class="container" >
+        <div class="row text-center" >
+            <div class="col-sm-4" style="height:350px">
                 <h2 class="number">Phimp.me</h2>
                 <p>
                     Do you have many different apps in your Android Phone for the purpose of clicking, editing and sharing images? Hey guess what, now you don't need all of those because Phimp.me is here, with all the features packaged into one. So go and clean your phone memory to save your space for other useful purposes.
                 </p>                
-                <div class="row" style="margin-top:75px">
+                <div class="row" style="position:absolute; bottom:0px;left:30%">
                     <div class="col-sm-12"><a class="btn" href="https://phimp.me">Learn More</a></div>
                 </div>                
             </div>            
-            <div class="col-sm-4">
+            <div class="col-sm-4" style="height:350px">
                 <h2 class="number">PSLab Android</h2>
                 <p>
                     PSLab is a tiny pocket science lab that provides an array of equipment for doing science and engineering experiments. It can function like an oscilloscope, waveform generator, frequency counter, programmable voltage and current source and also as a data logger.
                 </p>                
-                <div class="row" style="margin-top:95px">
+                <div class="row" style="position:absolute; bottom:0px; left:30%">
                     <div class="col-sm-12"><a class="btn" href="http://pslab.fossasia.org">Learn More</a></div>
                 </div>           
             </div>
-            <div class="col-sm-4">
+            <div class="col-sm-4" style="height:350px">
                 <h2>Open Event Organizer App</h2>
                 <p>Open Event Organizer App provides event management features like:<br>
 					- Event manager log in and sign up<br>
 					- Manual attendee check in<br>
 					- Attendee check in through QR code scan<br>
 					- Basic event analytics
-                </p>
-                <div class="row" style="margin-top:5px">
+                </p>                
```

Comment:
Remove extra spacing.

Reply:
They were like that. I didn't change them

Reply:
Ah, my bad, looks cool. thanks